README.md

```diff
@@ -30,5 +30,38 @@ config :logger, :console,
 
 ```
 
+## Configuration
+
+You can add metadata to all your requests via config options:
+
+```elixir
+config :cee_log_formatter,
+  metadata: [
+    app_name: "my-app",
+    arbitrary_mfa: {MyMod, :some_func, [:arg1, :arg2]}
+  ]
+```
+
+The `metadata` config takes a keyword list. If the value is a string, it is used directly in the
+log output. If the value is a 3-element tuple, it is expected to be a `{Module, function,
+argument-list}`, which is called for every log line to get some current value. If the value
+returned from the MFA is not a string, it is ignored.
```

Comment:
PRO TIP: to auto-format Markdown files (including wrapping lines at 80 characters), use this shell function:
```shell
mdf() {
  local f="${1:-README.md}"
  prettier --prose-wrap=always --print-width=80 "${f}" | sponge "${f}"
}
```
Requires Prettier () and `sponge`.

Reply:
Cool. I actually have my editor set to wrap my markdown at 100 columns, which I prefer to 80 :)

Reply:
I like 80 to avoid ugly line wraps in PRs like this ^^^